src/node/desktop/src/main/gwt-callback.ts

```diff
@@ -544,6 +544,7 @@ export class GwtCallback extends EventEmitter {
 
     ipcMain.on('desktop_bring_main_frame_to_front', () => {
       this.mainWindow.window.focus();
+      console.debug('desktop_bring_main_frame_to_front');
```

Comment:
Did you mean to leave this in?

Reply:
I'll clean this up. I got side tracked into trying to reproduce another bug!